test/src/bfcache.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('BFCache', function () {
     });
 
     try {
-      page.setDefaultTimeout(2000);
+      page.setDefaultTimeout(5000);
```

Comment:
Why this change?

Reply:
Tests were timing out here.

Reply:
2 sec should be more than enough for anything in this test unless smth else is going wrong or the host is not healthy

Reply:
it seems it timed out with 5 sec too